File: backend/components/metiers/threed/tests_centralises/fixtures/helpers/audit/test_helper_audit.py
# test_helper_audit.py – Tests helpers audit (Python)
import importlib.util
import os

spec = importlib.util.spec_from_file_location(
    "helper_audit",
    "/workspaces/dihya.io/backend/components/metiers/threed/fixtures/helpers/audit/helper_audit.py",
)
helper_audit = importlib.util.module_from_spec(spec)
spec.loader.exec_module(helper_audit)


def test_log_audit_event(capsys):
    helper_audit.log_audit_event("event")
    captured = capsys.readouterr()
    assert "AUDIT: event" in captured.out

File: backend/components/metiers/threed/tests_centralises/fixtures/helpers/audit/test_helper_audit.js
// test_helper_audit.js – Tests helpers audit (JS)
const { logAuditEvent } = require('../../../../fixtures/helpers/audit/helper_audit');
test('logAuditEvent', () => {
  console.log = jest.fn();
  logAuditEvent('event');
  expect(console.log).toHaveBeenCalledWith('AUDIT: event');
});
